server/nova-components/CreateFavorite/resources/js/components/Tool.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div>
+        <Form title="Favorite Creation"/>
+
+    </div>
+</template>
+
+<script>
+import Form from './form/form.vue'
```

Comment:
Register your components in nova-components/'YourToolName'/resources/js/tool.js and use it here